src/nprintml/net/step.py

```diff
@@ -132,18 +133,18 @@ def __init__(self, parser):
         )
         self.group_parser.add_argument(
             '-P', '--pcap-file', '--pcap_file',
-            default=(),
+            action='append',
+            default=[],
             metavar='FILE',
-            nargs='*',
```

Comment:
This was a mistaken configuration which would now conflict with subcommand(s) – (these arguments definitely don't take `*` *i.e.* arbitrary number of values).